### PR TITLE
Publishes Production Schema from CI

### PR DESCRIPTION
**Pull request by redreceipt**
**IMPORTANT: If this is for testing code in our npm dependencies, you should be pointing your PR at the `canary` or `next` branch. `master` should never rely on unreleased code.**

### CODE REVIEW

**Comment by redreceipt** [comment]
need to provide GET query to get back a 200 somehow on the healthcheck

File: apollos-church-api/src/server.js [hunk 1]
```diff
@@ -60,6 +60,11 @@ const apolloServer = new ApolloServer({
 
 const app = express();
 
+// health check
+app.get('/health', (req, res) => {
+  res.send('ok');
+});
+
```
Comment (redreceipt) [inline apollos-church-api/src/server.js:67]: @vinnyjth what do you think about this? simple and elegant. and I constantly wanted something like this at newspring. main purpose is to validate the deploy.

Comment (redreceipt) [reply apollos-church-api/src/server.js:67]: @vinnyjth you see this?

Comment (vinnyjth) [reply apollos-church-api/src/server.js:67]: Thanks for the ping, this is good!

**Review by vinnyjth — APPROVED** [review]

**Pull request merged**